WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -57,6 +58,7 @@
     private Map<String, View> mUsernameErrorViews = new Hashtable<>();
     private String mRole;
     private String mCustomMessage = "";
+    private boolean mInviteOperationInProgress = false;
```

Comment:
This seems like an addition to the Fragment's state so, I suspect it needs to be retained on rotation (via `onSaveInstanceState()`). Ignore this if saving is unneeded. Thanks!